Negotiation Mastery Program Merge Try 2

Computer8543 opened the pull request:
I fixed the issues with the Negotiation Mastery Program YAML file you brought up, and it is much closer to being ready to merge. Are there any more improvements you would like to suggest be made to the YAML file?

billwestfall commented:
This is a good topic, but not sure I completely grok the critical fourth person perspective. Maybe a topic for a Circle call coming up?

michael-basil commented:
It's a super stretch topic for sure.

I am curious what others may think about it.

Circle calls have an "energy" feeling, that's what I think it seeks to.

jimmanzzullo commented:
It's like when Luke goes to Dagobah to find Yoda and begins his training.  He finally starts to find his fourth person perspective just before he leaves.  (sorry for SW reference :) ). 

Gonzo Circle :tm: ~Jimmy

Computer8543 committed:
Add negotiation-mastery-program.yaml

Computer8543 committed:
Move negotiation-mastery-program.yaml

Computer8543 committed:
Changed Principles from numbered list to label list.

Computer8543 committed:
Deleted continuous and belt reflection, run the gauntlet, and things in between those two.

Computer8543 committed:
Changed belt levels attribute to levels

Computer8543 committed:
Changed White Belt Principle from Organization to Focus.

Computer8543 committed:
Fixed White Belt Kyu

Computer8543 committed:
Fixed Yellow Belt Kyu

Computer8543 committed:
Fixed Green Belt Kyu

Computer8543 committed:
Fixed Blue Belt Kyu

Computer8543 committed:
Fixed Purple Belt Kyu

Computer8543 committed:
Fixed Brown Belt Kyu

Computer8543 committed:
Fixed Red Belt Kyu

Computer8543 committed:
Changed Black Belt Kyu attribute to Shodan, and set value at 1 for 1st Shodan.

Computer8543 committed:
Move negotiation-mastery-program.yaml from _data folder to root folder.

Computer8543 committed:
Update and rename negotiation-mastery-program.yaml to program.yaml

Changed name of Negotiation Mastery Program YAML file to make it simpler and easier to search for.

Computer8543 committed:
Move forms attribute and sub attributes out of reflection attribute

Computer8543 committed:
Added Continuous and Belt strings to Reflection attribute.

Computer8543 committed:
Move community attribute to be a sibling attribute after awareness.

Computer8543 committed:
Changed White Belt principle attribute value from Focus to Orientation.

Computer8543 committed:
Moved white sub-attribute in levels attribute 2 spaces in.

Computer8543 committed:
Moved yellow sub-attribute in levels attribute 2 spaces in.

Computer8543 committed:
Moved green sub-attribute in levels attribute 2 spaces in.

Computer8543 committed:
Moved blue sub-attribute in levels attribute 2 spaces in.

Computer8543 committed:
Moved purple sub-attribute in levels attribute 2 spaces in.

Computer8543 committed:
Moved brown sub-attribute in levels attribute 2 spaces in.

Computer8543 committed:
Moved red sub-attribute in levels attribute 2 spaces in.

Computer8543 committed:
Moved black sub-attribute in levels attribute 2 spaces in.

Computer8543 committed:
Make levels attribute a new document within the program.yaml file.

Computer8543 committed:
Fixed spacing issue between community attribute and levels document.

Computer8543 committed:
Added comment below the 3 dashes signifying a new document for the levels document.

Computer8543 committed:
Remove black attribute comment in labels document.

Computer8543 committed:
Changed principle subattribute in white subattribute of levels attribute to focus subattribute.

Computer8543 committed:
Changed principle attribute of yellow belt to focus attribute.

Computer8543 committed:
Changed principle attribute of green belt to focus attribute.

Computer8543 committed:
Changed principle attribute of blue belt to focus attribute.

Computer8543 committed:
Changed principle attribute of purple belt to focus attribute.

Computer8543 committed:
Changed principle attribute of brown belt to focus attribute.

Computer8543 committed:
Changed principle attribute of red belt to focus attribute.

Computer8543 committed:
Changed principle attribute of black belt to focus attribute.

Computer8543 committed:
Changed white attribute to white value of color attribute.

Computer8543 committed:
Changed yellow attribute to yellow value of color attribute.

Computer8543 committed:
Changed green attribute to green value of color attribute.

Computer8543 committed:
Changed blue attribute to blue value of color attribute.

Computer8543 committed:
Changed purple attribute to purple value of color attribute.

Computer8543 committed:
Changed brown attribute to brown value of color attribute.

Computer8543 committed:
Changed red attribute to red value of color attribute.

Computer8543 committed:
Changed black attribute to black value of color attribute.

Computer8543 committed:
Unite color, kyu, focus, challenge, and exercises attributes for white belt into a single object.

Computer8543 committed:
Unite color, kyu, focus, challenge, and exercises attributes for yellow belt into a single object.

Computer8543 committed:
Unite color, kyu, focus, challenge, and exercises attributes for green belt into a single object.

Computer8543 committed:
Unite color, kyu, focus, challenge, and exercises attributes for blue belt into a single object.

Computer8543 committed:
Unite color, kyu, focus, challenge, and exercises attributes for purple belt into a single object and fixed subattribute spacing.

Computer8543 committed:
Unite color, kyu, focus, challenge, and exercises attributes for brown belt into a single object and fixed subattribute spacing.

Computer8543 committed:
Unite color, kyu, focus, challenge, and exercises attributes for purple belt into a single object and fixed subattribute spacing.

Computer8543 committed:
Unite color, kyu, focus, challenge, and exercises attributes for red belt into a single object and fixed subattribute spacing.

Computer8543 committed:
Unite color, shodan, focus, challenge, and exercises attributes for black belt into a single object and fixed subattribute spacing.

Computer8543 committed:
Change kyu attribute for white belt into level attribute.

Computer8543 committed:
Change kyu attribute for yellow belt into level attribute.

Computer8543 committed:
Change kyu attribute for green belt into level attribute.

Computer8543 committed:
Change kyu attribute for blue belt into level attribute.

Computer8543 committed:
Change kyu attribute for purple belt into level attribute.

Computer8543 committed:
Change kyu attribute for brown belt into level attribute.

Computer8543 committed:
Change kyu attribute for red belt into level attribute.

Computer8543 committed:
Change shodan attribute for black belt into level attribute.

Computer8543 committed:
Fixed exercises subattributes spacing for white level.

Computer8543 committed:
Fixed exercises subattributes spacing for yellow level.

Computer8543 committed:
Fixed exercises subattributes spacing for green level.

Computer8543 committed:
Fixed exercises subattributes spacing for blue level.

Computer8543 committed:
Unite white level exercise subattributes label and url into exercise sub-objects of the level object.

Computer8543 committed:
Unite yellow level exercise subattributes label and url into exercise sub-objects of the level object.

Computer8543 committed:
Unite green level exercise subattributes label and url into exercise sub-objects of the level object.

Computer8543 committed:
Unite blue level exercise subattributes label and url into exercise sub-objects of the level object.

Computer8543 committed:
Unite purple level exercise subattributes label and url into exercise sub-objects of the level object.

Computer8543 committed:
Unite brown level challenge subattributes label and url into challenge sub-object of the level object.

Computer8543 committed:
Unite brown level exercise subattributes label and url into exercise sub-object of the level object.

Computer8543 committed:
Unite red level exercise subattributes label and url into exercise sub-objects of the level object.

Computer8543 committed:
Unite black level exercise subattributes label and url into exercise sub-objects of the level object.

Computer8543 committed:
Changed green level challenge Analyze ... subattribute into a value by removing the colon at the end.

Computer8543 committed:
Changed negotiation 9 moves attribute in blue level into label attribute with Negotiation 9 Moves value.

Computer8543 committed:
Changed negotiation 9 moves attribute in purple level into label attribute with Negotiation 9 Moves value.

Computer8543 committed:
Changed negotiation 9 moves attribute in brown level into label attribute with Negotiation 9 Moves value.

Computer8543 committed:
Changed negotiation 9 moves attribute in red level into label attribute with Negotiation 9 Moves value.

Computer8543 committed:
Changed zen leader flips attribute in blue level into label attribute with Zen Leader Flips value.

Computer8543 committed:
Changed zen leader flips attribute in purple level into label attribute with Zen Leader Flips value.

Computer8543 committed:
Changed zen leader flips attribute in brown level into label attribute with Zen Leader Flips value.

Computer8543 committed:
Changed zen leader flips attribute in red level into label attribute with Zen Leader Flips value.

Computer8543 committed:
Added Negotiation 9 moves url to label object in blue level.

Computer8543 committed:
Added Negotiation 9 Moves url to label object in purple level.

Computer8543 committed:
Added Negotiation 9 Moves url to label object in brown level.

Computer8543 committed:
Added Negotiation 9 Moves url to label object in red level.

Computer8543 committed:
Remove blue belt Negotiation 9 Moves URL.

Computer8543 committed:
Remove purple belt Negotiation 9 Moves URL.

Computer8543 committed:
Remove brown belt Negotiation 9 Moves URL.

Computer8543 committed:
Remove red belt Negotiation 9 Moves URL.

Computer8543 committed:
Made Negotiation 9 Moves (Dynamic) string under forms in practices into a value of a label.

Computer8543 committed:
Made Zen Leader Essential Flips (Static) string under forms in practices into a value of a label.

Computer8543 committed:
Added URL to Negotiation 9 Moves (Dynamic) form object.

Computer8543 committed:
Added URL attribute to Zen Leader Essential Flips (Static) forms object.

Computer8543 committed:
Change Have ... attribute into value in red level exercises.

Computer8543 committed:
Change Have ... attribute in black level into value.

Computer8543 committed:
Turn Have a discussion ... value in red level into a value of a label.

Computer8543 committed:
Turn Have a discussion ... value in black level into a value of a label.

Computer8543 committed:
Add colon and enclose label value in quotation marks of Have a discussion... exercise label in black level.

Computer8543 committed:
Add colon and enclose label value in quotation marks of Have a discussion... exercise label in red level.

Computer8543 committed:
Added discussion_points attribute to black level exercises.

Computer8543 committed:
Added discussion_points attribute to red level exercises.

Computer8543 committed:
Make green level challenge Analyze a spirited ... value a value of a label.

Computer8543 committed:
Enclosed Analyze a spirited... label value in green level challenge in quotation marks and added a colon to the end of the sentence.

Computer8543 committed:
Added challenge_points attribute to green level challenge.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+        - label: Atomic Habits - How to Get 1% Better Every Day
+          url: https://www.youtube.com/watch?v=U_nzqnXWvSo
+        - label: "Life Without ‘Should’: A Simple Practice for Deeper Awareness 💡"
+          url: https://www.linkedin.com/posts/michaelrbasil_emotionalintelligence-selfawareness-mindset-activity-7270106763566166017-Xb1c?utm_source=share&utm_medium=member_android&rcm=ACoAAETrLT4B27LcbnDr0JNGm-Ngj9HoidcoxKs
```
Remove the tracking code (after the ?)

Computer8543 replied:
I removed the tracking code.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+          url: https://m.youtube.com/watch?v=3-wjmIFlnNo
+        - label: Distilling Conflation
+          url: https://www.linkedin.com/posts/michaelrbasil_mindset-dojo-reflection-activity-7310643830255456256-jwyT?utm_source=share&utm_medium=member_ios&rcm=ACoAAABROmsBfSucj7w-d4OF1-a1oj5GPId0Jq8
+        - label: Negotiation 9 Moves    
```
The child bullets are incorrect YAML ... flatten these like a cross product ... for the URL on each move, use the same URL ... the link to the negotiation 9 in forms

Computer8543 replied:
I made the Negotiation 9 moves into a cross-product and added the URL to each of the moves.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+          url: https://www.linkedin.com/posts/michaelrbasil_breaking-the-cycle-from-intellectual-burnout-activity-7247773142536859648-dUec?utm_source=share&utm_medium=member_ios&rcm=ACoAAETrLT4B27LcbnDr0JNGm-Ngj9HoidcoxKs
+        - label: A neuroscientist’s guide to fixing your mind in 15 mins
+          url: https://bigthink.com/series/the-big-think-interview/anne-laure-mindsets/
+        - label: Negotiation 9 Moves
```
The child bullets are incorrect YAML ... flatten these like a cross product ... for the URL on each move, use the same URL ... the link to the negotiation 9 in forms

Computer8543 replied:
Changed child bullets into cross product and added URL attribute to Negotiation 9 Moves objects.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+      exercises:
+        - label: Control vs Adjustment
+          url: https://www.youtube.com/shorts/2erEr5grqAk
+        - label: Negotiation 9 Moves
```
The child bullets are incorrect YAML ... flatten these like a cross product ... for the URL on each move, use the same URL ... the link to the negotiation 9 in forms

Computer8543 replied:
Flattened child bullets like a cross product, but we might have to fine-tune the naming of the object attributes to be machine readable.

michael-basil replied:
From a rendering perspective it will be easier to keep a label / url pairing convention ...

So instead of sublabel make it like this...

`label: Negotiation` 9 Move - "I" Messages'

Notice I dropped the s on moves.

This will optimize the rendering to be simpler ... showing my bis!

Computer8543 replied:
Thank you for explaining that. I will apply that pairing convention to this and the other instances you pointed out in this document.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+      challenge: 
+        - Utilize all Negotiation 9 Moves and some of the Zen Leader Flips in Negotiation.
+      exercises:
+        - label: Negotiation 9 Moves
```
The child bullets are incorrect YAML ... flatten these like a cross product ... for the URL on each move, use the same URL ... the link to the negotiation 9 in forms

Computer8543 replied:
Incorporated suggestion.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+          - Labels
+          - Mirrors
+          - Dynamic Silence
+        - label: Zen Leader Flips
```
The child bullets are incorrect YAML ... flatten these like a cross product ... for the URL on each move, use the same URL ... the link to the Zen Leader Flips in forms

Computer8543 replied:
I flattened the child bullets into a cross product and added the URL to each Zen Leader Flip.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+          - Paraphrasing
+          - Summary
+          - Calibrated Questions
+        - label: Zen Leader Flips
```
The child bullets are incorrect YAML ... flatten these like a cross product ... for the URL on each move, use the same URL ... the link to the Zen Leader Flips in forms

Computer8543 replied:
Flattened child bullets into cross-product and added URL attribute and value to cross-product objects.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+        - label: Negotiation 9 Moves
+          - “I” Messages
+          - Encouragers
+        - label: Zen Leader Flips
```
The child bullets are incorrect YAML ... flatten these like a cross product ... for the URL on each move, use the same URL ... the link to the Zen Leader Flips in forms

Computer8543 replied:
Implemented suggestion.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+      exercises:
+        - label: Negotiation 9 Moves
+          - Tone
+        - label: Zen Leader Flips
```
The child bullets are incorrect YAML ... flatten these like a cross product ... for the URL on each move, use the same URL ... the link to the Zen Leader Flips in forms

Computer8543 replied:
Incorporated suggestion.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+        - label: What if He Falls? The Terrifying Reality Behind Filming “Free Solo”
+          url: https://m.youtube.com/watch?v=3-wjmIFlnNo
+        - label: Distilling Conflation
+          url: https://www.linkedin.com/posts/michaelrbasil_mindset-dojo-reflection-activity-7310643830255456256-jwyT?utm_source=share&utm_medium=member_ios&rcm=ACoAAABROmsBfSucj7w-d4OF1-a1oj5GPId0Jq8
```
remove tracking code on the URL - past ?

Computer8543 replied:
I removed the tracking code on the URL.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+        - TBD
+      exercises:
+        - label: Why Did Agile Fail?
+          url: https://www.linkedin.com/posts/michaelrbasil_mindset-agile-dojo-activity-7260441419134246916-I_1u?utm_source=share&utm_medium=member_android&rcm=ACoAAETrLT4B27LcbnDr0JNGm-Ngj9HoidcoxKs
```
remove tracking code on the URL - past ?

Computer8543 replied:
Removed tracking code.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+        - label: Why Did Agile Fail?
+          url: https://www.linkedin.com/posts/michaelrbasil_mindset-agile-dojo-activity-7260441419134246916-I_1u?utm_source=share&utm_medium=member_android&rcm=ACoAAETrLT4B27LcbnDr0JNGm-Ngj9HoidcoxKs
+        - label: "Breaking the Cycle: From Intellectual Burnout to Integrated Flow"
+          url: https://www.linkedin.com/posts/michaelrbasil_breaking-the-cycle-from-intellectual-burnout-activity-7247773142536859648-dUec?utm_source=share&utm_medium=member_ios&rcm=ACoAAETrLT4B27LcbnDr0JNGm-Ngj9HoidcoxKs
```
remove tracking code on the URL - past ?

Computer8543 replied:
Removed tracking code.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+        - label: Zen Leader Flips
+          - Local Self to Whole Self
+          - Delusion to Awakening 
+        - label: "Have a discussion with 3 people individually and then in a group about:"
```
flatten these into a cross product ... label and URL pairings

Computer8543 replied:
Implemented suggestion.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+      exercises:
+        - label: The Backwards Brain Tractor
+          url: https://youtu.be/lylCYkgC63Q?si=FzuWfAHq4Yie_AHC
+        - label: "Have a discussion with 3 people individually and then in a group about:"
```
flatten these into a cross product ... label and URL pairings

Computer8543 replied:
Implemented suggestion.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,169 @@
+  - label: Form reflection into reflex
+practices:
+  - reflection:
+    - Continuous
```
for balance denote this with an attribute of: `label: Continuous` and `label: Belt`

Computer8543 replied:
Made suggested change.

Computer8543 committed:
Removed tracking code from Life Without Should ... LinkedIn Post URL.

Computer8543 committed:
Removed tracking code on Distilling Conflation article URL.

Computer8543 committed:
Made reflection strings under practices into label values.

Computer8543 committed:
Flattened Negotiation 9 Moves child bullets in brown level like a cross product.

Computer8543 committed:
Added URL to Negotiation 9 Moves objects under brown level exercises.

Computer8543 committed:
Changed move attribute to sublabel attribute for Negotiation 9 Moves in brown level for standardization purposes.

Computer8543 committed:
Removed tracking code on "Why Did Agile Fail?" article URL.

Computer8543 committed:
Removed tracking code on Breaking the Cycle ... article URL.

Computer8543 committed:
Changed child bullets in blue level Negotiation 9 Moves into cross product.

Computer8543 committed:
Added URL to Negotiation 9 Moves cross-product in blue level.

Computer8543 committed:
Made Zen Leader Flips child bullets in blue level into cross-product.

Computer8543 committed:
Merged Zen Leader Flips sublabels and labels in blue level into a single object for each flip.

Computer8543 committed:
Added URL to each Zen Leader Flip object in blue level.

Computer8543 committed:
Enclosed purple exercise object label attribute A neuroscientist's guide ... in quotation marks.

Computer8543 committed:
Changed Negotiation 9 Moves child bullets in purple level into cross-product.

Computer8543 committed:
Added URL attribute to Negotiation 9 Moves objects in purple level.

Computer8543 committed:
Changed Zen Leader Flips child bullets in purple level into cross-product.

Computer8543 committed:
Added URL attribute and value to Zen Leader Flips objects in purple level.

Computer8543 committed:
Simplified rendering by incorpating sublabel content into label attribute for Negotiation 9 Moves in blue level.

Computer8543 committed:
Incorporated sublabel value into label value and removed sublabel attribute for Zen Leader Essential Flip objects in blue level to simplify rendering.

Computer8543 committed:
Incorporated sublabel value into label value and removed sublabel attribute for Negotiation 9 Move objects in purple level to simplify rendering.

Computer8543 committed:
Incorporated sublabel value into label value and removed sublabel attribute for Zen Leader Essential Flip objects in purple level to simplify rendering.

Computer8543 committed:
Incorporated sublabel value into label value and removed sublabel attribute for Negotiation 9 Move objects in brown level to simplify rendering.

Computer8543 committed:
Change child bullets of Zen Leader Flips in brown level into cross-product label attributes.

Computer8543 committed:
Added URL attribute and value to Zen Leader Essential Flip objects in brown level.

Computer8543 committed:
Incorporated child bullet into label for Negotiation 9 Move object in red level.

Computer8543 committed:
Added URL attribute and value to Negotiation 9 Move object in red level.

Computer8543 committed:
Changed red level challenge to incorporate the word Essential between Zen Leader and Flips.

Computer8543 committed:
Incorporated child bullets into label attribute value for Zen Leader Essential Flip objects in red level.

Computer8543 committed:
Added URL attribute and value to Zen Leader Essential Flip objects in red level.

Computer8543 committed:
Flattened two seperate label lists about Have a discussion ... into two cross product objects in red level.

Computer8543 committed:
Changed two seperate lists about Have a discussion ... into two different cross-product objects in black level.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,173 @@
+  - Metaphorical
+community:
+  - url: https://dojo.center/community
+---
```
Remove the separation ... this is all the same document.

Remove --- and the comment for levels document

Computer8543 replied:
Implemented suggestion.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,173 @@
+levels:
+    - color: white
+      level: -7
+      focus: Orientation
```
Add an attribute ... `label: 7th Kyu`

Apply the same to each of the other kyu ranks

Computer8543 replied:
Implemented suggestion.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,173 @@
+          url: https://a.co/d/2vxaK20
+    - color: black
+      level: 1
+      focus: "None given"
```
`focus: Wu-Wei`

Computer8543 replied:
Implemented suggestion.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,173 @@
+        - label: Have a discussion with 3 people individually and then in a group about - Never Split the Difference
+          url: https://a.co/d/2vxaK20
+    - color: black
+      level: 1
```
Add `label: Shodan`

Computer8543 replied:
Implemented suggestion.

Computer8543 committed:
Merge levels document with prior document into one document.

Computer8543 committed:
Added label attribute to white level object.

Computer8543 committed:
Added label attribute to yellow level object.

Computer8543 committed:
Added label attribute to green level object.

Computer8543 committed:
Added label attribute to blue level object.

Computer8543 committed:
Added label attribute to purple level object.

Computer8543 committed:
Added label attribute to brown level object.

Computer8543 committed:
Added label attribute to red level object.

Computer8543 committed:
Added label attribute to black level object.

Computer8543 committed:
Changed black level focus attribute value from "None Given" to Wu-Wei.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,179 @@
+      level: -5
+      focus: Perception
+      challenge: 
+        - label: "Analyze a spirited conversation from the following models of understanding:"
```
Remove the challenge_points attribute and instead simplify this to ...

Analyze a spirited conversation through the FEBI, Cynefin and Map of Consciousness lenses.

Computer8543 committed:
Made two green level challenge lists into cross-product to simplify rendering.

Computer8543 committed:
Remove label attribute and make values into strings for green level challenge cross-product.

Computer8543 commented on program.yaml:
```diff
@@ -0,0 +1,177 @@
+      label: 5th Kyu
+      level: -5
+      focus: Perception
+      challenge: 
```
Made two lists into cross-product, and removed label attribute to make three strings.

michael-basil replied:
For this one ... a cross product is not needed.

Collapse into one item ... 

Analyze a spirted conversation through the lenses of FEBI, Cynefin and Map of Consciousness.

Computer8543 replied:
Merged three strings into one value of challenge attribute.

Computer8543 committed:
Merge 3 green level challenge bullet points into one.

Computer8543 committed:
Make green level challenge a attribute instead of a bullet.

Computer8543 committed:
Make blue level challenge a attribute instead of a bullet.

Computer8543 committed:
Make purple level challenge a attribute instead of a bullet.

Computer8543 committed:
Replace Control vs Adjustment in brown level with Breathing to Support the FEBI Patterns and make challenge TBD.

Computer8543 committed:
Make red level challenge bullet point into an attribute value.

Computer8543 committed:
Get rid of all other challenges in black level other than Negotiation Randori.

Computer8543 committed:
Resolved inconsistancy in awareness attribute with principles attribute by making each bullet in the awareness list a label attribute like in principles attribute.

Computer8543 commented on program.yaml:
```diff
@@ -0,0 +1,166 @@
+      url: https://www.blackswanltd.com/newsletter/infographic-the-black-swan-groups-negotiation-9
+    - label: Zen Leader Essential Flips (Static)
+      url: https://cdn.dirigible.studio/wp-content/uploads/sites/24/2021/09/04073619/TZL-graphic-izl.pdf
+awareness:
```
Fixed inconsistancy in awareness bullet list with principles bullet list by making each bullet point a label attribute.

Computer8543 committed:
Remove periods from end of challenge attribute value sentences to simplify rendering.

michael-basil commented on program.yaml:
```diff
@@ -0,0 +1,166 @@
+title: Negotiation Mastery Program
+mission: Awaken the negotiator within—calm, clear, and creative.
```
Remove the period

michael-basil committed:
fix: Remove extra period

michael-basil reviewed (approved):
Nice work @Computer8543
